plugins/outputs/azure_data_explorer/azure_data_explorer.go

```diff
@@ -37,6 +37,8 @@ type AzureDataExplorer struct {
 const (
 	tablePerMetric = "tablepermetric"
 	singleTable    = "singletable"
+	// These control the amount of memory we use when ingesting blobs
+	azureBlobBlockSize = 1 * 1024 * 1024
```

Comment:
Name is too long.  It is clear this is for azure from the package name, blob storage is hidden so we don't need to reference it.

This should simply be 
```go
blockSize = 1024 * 1024 // 1 MiB
```
If you want more clarity, I'd probably do:
```go
_1MiB = 1 << 20
blockSize = 1 * 1MiB
```
The "_" is simply because you can't start a Go variable name with a number. MiB to be clear we aren't rounding the value.

Reply:
done